### ISSUE TITLE
feat: Add support for `webpack_path` in ERB templates

### ISSUE THREAD

**Issue by MikeRogers0** [bridgetownrb/bridgetown#100]
Currently if you use ERB to to generate your layouts & call the `webpack_path` method like:

```erb
<script src="<%= webpack_path "js" %>" defer></script>
```

The user will see the exception:

```
undefined method `webpack_path' for #<Bridgetown::ERBView:0x000055947211fd08>
noMethodError: undefined method `webpack_path' for #<Bridgetown::ERBView:0x000055947211fd08>
  /usr/local/bundle/bundler/gems/bridgetown-20edf2cb06de/bridgetown-core/lib/bridgetown-core/ruby_template_view.rb:53:in `method_missing'
  /usr/src/app/src/_partials/_head.erb:4:in `__tilt_4320'
  /usr/local/bundle/gems/tilt-2.0.10/lib/tilt/template.rb:170:in `call'
  /usr/local/bundle/gems/tilt-2.0.10/lib/tilt/template.rb:170:in `evaluate'
  /usr/local/bundle/gems/tilt-2.0.10/lib/tilt/template.rb:109:in `render'
```

## Motivation

I'm a big fan of ERB of Liquid as it reminds me more of working with Rails, so ideally I'd prefer to do ERB in my portfolio if possible :)

## Unresolved Questions

We'd have to create a helper separately for ERB, which could be tricky. What is the best approach / where should this be added?

**Comment by jaredcwhite** [comment]
Should merge in #96 first…

**Comment by jaredcwhite** [comment]
Fixed in #105
